src/sage/symbolic/relation.py

```diff
@@ -929,12 +1022,12 @@ def solve(f, *args, **kwds):
 
         sage: # needs sage.libs.giac
         sage: solve([(2/3)^x-2], [x], algorithm='giac')
-        ...[[-log(2)/(log(3) - log(2))]]
+        [-log(2)/(log(3) - log(2))]
 
         sage: # needs sage.libs.giac
         sage: f = (sin(x) - 8*cos(x)*sin(x))*(sin(x)^2 + cos(x)) - (2*cos(x)*sin(x) - sin(x))*(-2*sin(x)^2 + 2*cos(x)^2 - cos(x))
         sage: solve(f, x, algorithm='giac')
-        ...[-2*arctan(sqrt(2)), 0, 2*arctan(sqrt(2)), pi]
+        [-2*arctan(sqrt(2)), 0, 2*arctan(sqrt(2)), pi]
```

Comment:
Presumably the `...` was meant to catch warnings, now the warning is fixed so they're safe to delete